README.md

```diff
@@ -19,7 +19,7 @@ Download Sync Gateway and other Couchbase packages for Linux, Windows and macOS
 
 To build Sync Gateway from source, you must have the following installed:
 
-* Go 1.17.5 or later.
+* Go 1.17.11 or later.
```

Comment:
Nitpick: We could relax this to any 1.17.x version.
There's no new APIs in the minor releases that we depend on.

```suggestion
* Go 1.17 or later.
```